### ISSUE TITLE
Low accuracy of OfficeHome dataset

### ISSUE THREAD

**Issue by TL-Wucw** [Albert0147/LPA_SFDA#1]
Thanks to the author for contributing the code!
Based on the hyperparameters provided in the paper, I found in my experiments that the accuracy of the OfficeHome dataset is very low (only 1.58% in task A->P) and seems difficult to fit. 

The source domain model parameters I used were provided by SHOT.

**Comment by Albert0147** [comment]
Hi. Please do not use other source model. Do use pytorch 1.3 with cuda 10.0 for training with our code. There is definitely something wrong, considering even source model can have better performance.

**Comment by TL-Wucw** [comment]
Thanks for your reply! 
Can you provide the following hyperparameters for the source model training on the Office31 and OfficeHome datasets?

**Comment by Albert0147** [comment]
Hi, I upload the unclean code for office-home and office31, I think the hyperparameters are the same in the paper.

**Comment by TL-Wucw** [comment]
Thanks!
